src/scripts/components/domBuilder.js

```diff
@@ -4,6 +4,7 @@ const domBuilder = () => {
   const domString = `
   <div id="navigation"></div>
   <div id="main-container">
+    <div id ="details-buttons"></div>
```

Comment:
I think this line is causing something funky to happen. If you navigate away from items, you'll still see the buttons on the page. 
![Screen Shot 2022-06-20 at 10 23 41 AM](https://user-images.githubusercontent.com/24661749/174634759-dd7aae96-f14b-4eab-a4c7-46f10b955319.png)